src/test/regress/expected/multi_insert_select.out

```diff
@@ -679,9 +679,10 @@ SELECT user_id, value_1_agg FROM agg_events ORDER BY 1,2;
        9 |          90
 (9 rows)
 
--- We do not support some CTEs
+-- We support CTEs
+BEGIN;
 WITH fist_table_agg AS
-  (SELECT sum(value_1) as v1_agg, user_id FROM raw_events_first GROUP BY user_id)
+  (SELECT max(value_1)+1 as v1_agg, user_id FROM raw_events_first GROUP BY user_id)
```

Comment:
The sum causes a primary key violations when there is only 1 (user_id, value_1), since the sum is the same as value_1 in tat case. By taking max + 1, we're always insert new values. Could have also done sum+1 I guess.